components/EventDetails.js

```diff
@@ -0,0 +1,67 @@
+import React, { PropTypes } from 'react';
+import { DateBlock } from '../components';
+import { Link } from '../routes';
+
+const renderAddress = ({ address_lines: addressLines, locality, region, postal_code: postalCode }) => {
+  // Will have to see how this data structure holds up over different events
+  return (
+    <div>
+      { addressLines[0] && <span>{addressLines[0]}</span> }
+      <span>{locality} {region}, {postalCode}</span>
+    </div>
+  );
+};
+
+const renderTimeRange = (startDate, endDate) => {
+  // TDB how to properly format this - would be nice to avoid bringing in moment.js
```

Comment:
I haven't used date-fns but some considerations with whatever library we choose should be

1.) File size -- moment and date-fns are > 60kb, so if we're only doing light date-manipulation it won't be worth the added weight
2.) Library popularity. Moment is more well-known and contributors jumping into this project would likely have an easier time using moment

I don't think we're at the point where we have to decide, just things to keep in mind